x-pack/legacy/plugins/uptime/public/pages/settings.tsx

```diff
@@ -0,0 +1,225 @@
+    }
+  };
+
+  const isFormDirty = dss.settings ? !isEqual(dss.settings, formFields) : true;
+
+  const headingText = i18n.translate('xpack.uptime.settingsPage.headerText', {
+    defaultMessage: 'Settings',
+    description: `The text that will be displayed in the app's heading when the settings page loads.`,
+  });
+
+  const breadcrumbs: ChromeBreadcrumb[] = [
+    {
+      text: 'Settings',
+    },
+  ];
+
+  return (
+    <>
+      <PageHeader headingText={headingText} breadcrumbs={breadcrumbs} datePicker={false} />
+      <form onSubmit={onApply}>
+        <EuiForm>
+          <EuiPanel>
+            <EuiTitle size="s">
+              <h3>
+                <FormattedMessage
+                  id="xpack.uptime.sourceConfiguration.indicesSectionTitle"
+                  defaultMessage="Indices"
+                />
+              </h3>
+            </EuiTitle>
+            <EuiSpacer size="m" />
+            <EuiDescribedFormGroup
+              title={
+                <h4>
+                  <FormattedMessage
+                    id="xpack.uptime.sourceConfiguration.heartbeatIndicesTitle"
+                    defaultMessage="Uptime indices"
+                  />
+                </h4>
+              }
+              description={
+                <FormattedMessage
+                  id="xpack.uptime.sourceConfiguration.heartbeatIndicesDescription"
+                  defaultMessage="Index pattern for matching indices that contain Heartbeat data"
+                />
+              }
+            >
+              <EuiFormRow
+                describedByIds={['heartbeatIndices']}
+                error={fieldErrors?.heartbeatIndices}
+                fullWidth
+                helpText={
+                  <FormattedMessage
+                    id="xpack.uptime.sourceConfiguration.heartbeatIndicesDefaultValue"
+                    defaultMessage="The default value is {defaultValue}"
+                    values={{
+                      defaultValue: <EuiCode>{defaultDynamicSettings.heartbeatIndices}</EuiCode>,
+                    }}
+                  />
+                }
+                isInvalid={!!fieldErrors?.heartbeatIndices}
+                label={
+                  <FormattedMessage
+                    id="xpack.uptime.sourceConfiguration.heartbeatIndicesLabel"
+                    defaultMessage="Heartbeat indices"
+                  />
+                }
+              >
+                <EuiFieldText
+                  data-test-subj="heartbeat-indices-input"
+                  fullWidth
+                  disabled={dss.loading}
+                  isLoading={dss.loading}
+                  value={formFields?.heartbeatIndices || ''}
+                  onChange={(event: any) =>
+                    onChangeFormField('heartbeatIndices', event.currentTarget.value)
+                  }
+                />
+              </EuiFormRow>
+            </EuiDescribedFormGroup>
+          </EuiPanel>
+
+          <EuiSpacer size="m" />
```

Comment:
Please move the <EuiSpacer> and <EuiFlexGroup> elements within the panel.
Example from APM
<img width="606" alt="Screenshot 2020-03-05 at 12 48 16" src="https://user-images.githubusercontent.com/20965076/75978827-a4927100-5edf-11ea-8cf8-7414d00051e0.png">

Reply:
Done!